tensorflow_datasets/core/dataset_info.py

```diff
@@ -407,6 +409,51 @@ def initialize_from_bucket(self):
       gcs_utils.download_gcs_file(fname, out_fname)
     self.read_from_directory(tmp_dir)
 
+  def show_examples(self, ds, rows=3, cols=3, plot_scale=3):
+    """Display some random (rows*columns) images from the image-dataset.
+    
+    This function will only work for image datasets which only have 
+    a single main image.
+    
+    Args:
+      ds: `tf.data.Dataset`. Does not accept batch data.
+      rows: `int`, number of rows of the display grid.
+      cols: `int`, number of columns of the display grid.
+      plot_scale: `int`, controls the plot size of the images. Keep this
+        value around 3 to get a good plot. High and low values may cause
+        the labels to get overlapped.
```

Comment:
Can you write an `Usage` example of this method?

Reply:
Yes, I will write in the documentation, once this is approved.

Reply:
Adding in the documentation is good, but having an example snippet in the docstring is also important for people reading the code.